openmc/tallies.py

```diff
@@ -2522,6 +2523,105 @@ def __neg__(self):
         new_tally = self * -1
         return new_tally
 
+
+    def get_values_slice(
+        self,
+        slice_index: float,
+        basis: str='xy',
+        score: typing.Optional[str] = None,
+        value: str = 'mean'
+    ):
+        """Gets a 2D array of values from a RegularMesh tally at specified index.
+
+        Parameters
+        ----------
+        slice_index : float
+            The axis index of the slice to extract.
+        basis : {'xy', 'xz', 'yz'}
+            The basis directions for the slice
+        score : str
+            A score string (e.g., 'absorption'). If score is left as None and
+            only one score is present in the tally then that score will be used
+        value : str
+            A string for the type of value to return  - 'mean' (default),
+            'std_dev', 'rel_err', 'sum', or 'sum_sq' are accepted
+
+        Returns
+        -------
+        np.ndarray
+            the 2D array of dataset values
+        """
+
+        # in this case there is only one score so we know which to slice even
+        if score == None and len(self.scores) == 1:
+            score = self.scores[0]
+        
+        tally = self.get_slice(scores=[score])
+        # get_values
+        data = tally.get_reshaped_data(value=value,expand_dims=True).squeeze()
```

Comment:
```suggestion
        data = tally.get_reshaped_data(value=value,expand_dims=True)
```
I think this can be removed now, I need to add some tests for the new tally methods first

Reply:
this can't be removed because ```get_slice(squeeze=True)``` does not squeeze the entire tally. It just squeezes the filters part